gnovm/cmd/gno/lint.go

```diff
@@ -96,42 +128,32 @@ func execLint(cfg *lintCfg, args []string, io commands.IO) error {
 				stdin, stdout, stderr,
 			)
 
-			targetPath := pkgPath
-			info, err := os.Stat(pkgPath)
-			if err == nil && !info.IsDir() {
-				targetPath = filepath.Dir(pkgPath)
+			memPkg := gno.MustReadMemPackage(pkgPath, pkgPath)
```

Comment:
Not from this PR either, but we might consider using `ReadMemPackage` without the `Must` here. I believe the lint command can handle the error properly and avoid panic.

Reply:
i'd say maybe for a time we remove the error handling here using `catchRuntimeError` ;)

Reply:
I ended up refactoring it anyway